### PR TITLE
VBID-10667 Support Expirations in the .NET Client

### PR DESCRIPTION
**Pull request by psimmons-vistar**
Adds a method to the Advertisement entity which allows for sending
an expiration. Refactored the method for sending PoP GET requests
to do so with minimal duplication.

Updates the tests to handle this case.

### CODE REVIEW

File: VistarClient/VistarClient/Entities/Advertisement.cs [hunk 1]
```diff
@@ -44,7 +44,19 @@ public Advertisement()
...
     public void SendProofOfPlay() {
-      var request = requestFactory.Create(ProofOfPlayUrl);
+      SendGetRequest(ProofOfPlayUrl);
+    }
+
+    public void SendProofOfPlay(ProofOfPlayMessage message) {
+      message.Send(ProofOfPlayUrl);
+    }
+
+    public void SendExpiration() {
```
Comment (psimmons-vistar) [inline VistarClient/VistarClient/Entities/Advertisement.cs]: Forgot to run command to update docs

**Comment by psimmons-vistar** [comment]
Meaningful changes in patch are in VistarClient/VistarClient/Entities/Advertisement.cs and VistarClient/VistarClient.Test/Entities/AdvertisementTest.cs

File: VistarClient/VistarClient/Entities/Advertisement.cs (hunk 1 above)
Comment (psimmons-vistar) [reply VistarClient/VistarClient/Entities/Advertisement.cs]: Done

**Comment by dmansen** [comment]
+1 looks good

**Pull request merged**